### PR TITLE
Feature/new app links

### PR DESCRIPTION
**Pull request by edbrett**
Changes external links to GFW flagship from alert emails:
- update `alert_link` to pass the correct data to the map link
- add new param for dashboards link to aoi

### CODE REVIEW

File: app/src/services/alertUrlService.js [hunk 1]
```diff
@@ -4,68 +4,53 @@ const BASE_URL = config.get('gfw.flagshipUrl');
...
-        if (subscription.params.geostore) {
-            query.geostore = subscription.params.geostore;
-        }
-        if (subscription.params.use) {
-            query.use = subscription.params.use;
-            query.useid = subscription.params.useid;
-            iso = 'ALL';
+        if (subscription.params.iso && subscription.params.iso.country) {
+          const country = subscription.params.iso.country;
+          const region = subscription.params.iso.region;
```
[the remark this answers is not in the document]
Comment (edbrett) [reply app/src/services/alertUrlService.js:15]: If you check the next line, I am conditionally adding this to the pathname depending on whether it is defined

File: app/src/services/alertUrlService.js [hunk 2]
```diff
@@ -4,68 +4,53 @@ const BASE_URL = config.get('gfw.flagshipUrl');
...
+          const subregion = subscription.params.iso.subregion;
```
[the remark this answers is not in the document]
Comment (edbrett) [reply app/src/services/alertUrlService.js:16]: https://github.com/gfw-api/gfw-subscription-api/pull/67/files/f0674e9cd9395e83dd31cb60c5d259f16f914d79#diff-899be98c51331e62b42801e4c79f4c55R17

File: app/src/services/alertUrlService.js [hunk 3]
```diff
@@ -4,68 +4,53 @@ const BASE_URL = config.get('gfw.flagshipUrl');
...
+                dataset: layer.datasetId,
+                layers: [dataset.layerId],
+                timelineParams: {
+                  startDate: moment(begin).format('YYYY-MM-DD'),
+                  endDate: moment(end).format('YYYY-MM-DD'),
+                  trimEndDate: moment(end).format('YYYY-MM-DD')
+                }
+              },
+              {
+                dataset: '0b0208b6-b424-4b57-984f-caddfa25ba22',
```
[the remark this answers is not in the document]
Comment (edbrett) [reply app/src/services/alertUrlService.js:43]: That makes sense. I will update the configs

File: app/src/services/alertUrlService.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (ikas) [reply app/src/services/alertUrlService.js:15]: that is absolutely right, sorry for pointing it out :)

**Comment by ikas** [comment]
LGTM :+1: 

Gonna add a commit to make the tests pass and we should be able to merge :)

**Pull request merged**